ipam/store.go

```diff
@@ -50,3 +51,47 @@ type IpamVm struct {
 	Id uint64 `sql:"AUTO_INCREMENT"`
 	//	IpamHostId sql.NullString
 }
+
+type ipamStore struct {
+	common.DbStore
+}
+
+func (ipamStore *ipamStore) addVm(stride uint, vm *Vm) error {
```

Comment:
Also, same comment as above with the "Vm" structure. The name should really be "Endpoint", instead.